test/lib/docker.test.ts

```diff
@@ -1,11 +1,32 @@
-import { readFileSync } from "fs";
+import * as crypto from "crypto";
+import { createReadStream, existsSync, readFileSync, unlinkSync } from "fs";
+import * as os from "os";
 import * as path from "path";
 import * as sinon from "sinon";
 import { test } from "tap";
 import * as subProcess from "../../lib/sub-process";
 
 import { Docker } from "../../lib/docker";
 
+function getChecksum(path: string): Promise<string> {
+  return new Promise((res, rej) => {
```

Comment:
let's go with resolve/reject, I think it's more readable
also "res" is some times used to abbreviate "response"